tests/cppunit/rwlock_test.cc

```diff
@@ -167,7 +167,7 @@ TEST(ReadWriteLock, WriteLockGurad_First) {
       });
     } else {
       ths[i] = std::thread([&rwlock, &val]() {
-        usleep(1000);  // To avoid it is the first to run, just sleep 100ms
+        usleep(100000);  // To avoid it is the first to run, just sleep 100ms
```

Comment:
```suggestion
        using namespace std::chrono_literals;
        std::this_thread::sleep_for(100ms);  // To avoid it is the first to run, just sleep 100ms
```

We can use the time literal `100ms` directly, haha : )